spec/spec_helper.rb

```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
+require 'simplecov'
+SimpleCov.start do
+  add_filter '/vendor/'
+  add_filter '/lib/myfile.rb'

```

Comment:
@f1nwe I agree with @igorkasyanchuk, why did you add ```'/lib/myfile.rb'``` ? and not just ```/lib```? Also are you sure that absolute path should be housed here?